public_html/wp-content/mu-plugins/blocks/assets/src/schedule/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * WordPress dependencies
+ */
+const { __ } = wp.i18n;
+
+/**
+ * Internal dependencies
+ */
+import { edit } from './edit';
+
+export const name  = 'wordcamp/schedule';
+export const LABEL = __( 'Schedule', 'wordcamporg' );
+export const ICON  = 'schedule';
+
+export const settings = {
+	title       : __( 'Schedule', 'wordcamporg' ),
+	description : __( "Display your WordCamp's awesome schedule.", 'wordcamporg' ),
+	icon        : ICON,
+	category    : 'wordcamp',
+	edit        : edit,
+	save        : () => null,
+
+	// todo keywordsto other blocks as well, in master and commit now
+
+	/*
+	 * Make the block full-width _only_ in the editor.
+	 *
+	 * It's very common for camps to have 3-4 tracks, but the editor's content area is very narrow, which makes it
+	 * difficult to read the schedule. Defining the block as full-width block gives it more space so it can be
+	 * easier to read, but we don't want to do that on the front-end, because that would unnecessarily add extra
+	 * styles that get in the way of organizer's customizing their CSS. It would also behave inconsistently between
+	 * themes that do and don't support the extra alignments that Gutenberg adds.
+	 *
+	 * Using the standard mechanism for full-width blocks would also add a toolbar icon which could be used to turn
+	 * off the wider view in the editor, which could cause confusion and clutter, without add benefit.
+	 *
+	 * Another potential solution would have been to manually add CSS rules to make the block full-width in the editor,
+	 * but that would have meant duplicating the existing rules from Core, and tightly coupling the editing experience,
+	 * which wouldn't be future-proof.
+	 *
+	 * Adding the alignment through `getEditWrapperProps()` allows us to achieve the desired result without any side-
+	 * effects.
+	 */
```

Comment:
Doesn't this break the concept of What-You-See-Is-What-You-Get? It seems misleading to me to have the schedule be inexplicably wider than all the rest of the content in the block editor, but the same width on the front end.

I would lean towards actually adding official support for wide and full alignments, like we just added for the other blocks. It seems like the schedule would have a similar use case, but also, the organizer could temporarily toggle it on if they need to get a better look at their schedule in the editor while also getting a more accurate WYSIWYG.

Reply:
🤔 

You make a good point about the inconsistency between the front and back ends.

I was thinking that was a there wasn't really a use case for turning off the full width, so it didn't make sense to use those options. Thinking about it again, though, I guess for single- and double-track camps, the default width would actually be fine, so maybe it would be better to use those, and let camps select wide/full when they have more tracks. I'll play around with it, thanks!